packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -1486,9 +1499,17 @@ class SliverOverlapInjector extends SingleChildRenderObjectWidget {
   const SliverOverlapInjector({
     Key key,
     @required this.handle,
+    @Deprecated(
+      'Use sliver instead. '
+      'This feature was deprecated after v1.10.16'
```

Comment:
```suggestion
      'This feature was deprecated after v1.10.16.'
```
I think the analyzer would prefer a fully punctuated sentence. 😝